src/search-injector/content_script.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './components/App'
+import * as constants from './constants'
+import { doSearch } from './actions'
+import { remoteFunction } from 'src/util/webextensionRPC'
+
+const fetchSearchInjection = remoteFunction('fetchSearchInjection')
+
+// Render Memex search results to the Google Search Page
+const getCmdMessageHandler = ({ cmd, ...payload }) => {
```

Comment:
This will all be much simpler once we get issue #back finished :) then just call a function rather than have the port setup.